ProfileService/ProfileService.BLL/Utilities/Messages/ProfileVisibilityMessages.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ProfileService.BLL.Utilities.Messages
+{
+    public static class ProfileVisibilityMessages
+    {
+        public const string VisibilityNotFound = "User profile visibility is not found";
+
+        public const string AccessDenied = "You can't access this visibility parameters";
+
+        public const string InvalidId = "User profile visibility options id is invalid";
+    }
+}
```

Comment:
it's better to have Resource files instead of such constants

Reply:
Replaced this constants with static methods